ui/options_advanced.ui

```diff
@@ -20,13 +20,65 @@
       <property name="spacing">
        <number>2</number>
       </property>
-      <item row="5" column="0">
-       <widget class="QCheckBox" name="recursively_add_files">
+      <item row="1" column="0">
+       <widget class="QLabel" name="label_ignore_regex">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
+        </property>
         <property name="text">
-         <string>Include sub-folders when adding files from folder</string>
+         <string>Ignore file paths matching the following regular expression:</string>
+        </property>
+        <property name="wordWrap">
+         <bool>true</bool>
+        </property>
+       </widget>
+      </item>
+      <item row="8" column="0">
+       <layout class="QHBoxLayout" name="horizontalLayout_2">
+        <item>
+         <widget class="QLabel" name="label_query_limit">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
+            <horstretch>0</horstretch>
+            <verstretch>0</verstretch>
+           </sizepolicy>
+          </property>
+          <property name="text">
+           <string>Max. number of entities to return per MusicBrainz query</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QSpinBox" name="query_limit">
+          <property name="accelerated">
+           <bool>true</bool>
+          </property>
+          <property name="minimum">
+           <number>1</number>
+          </property>
+          <property name="maximum">
+           <number>100</number>
```

Comment:
I'm not sure about those min/max values.
A value of 1 doesn't look very useful. What are the drawbacks when setting a value above 100?

I'd suggest we use a low value of 10 (it used to be 25), and add a tool tip to explain the drawbacks of setting the value too high or too low.

Also do we really need a granularity of 1 for this ? Not sure 30 or 31 do much difference. A step of 5 may be enough, dunno.

Reply:
Does MB have a maximum number that it will honor?  If so then that might be the maximum value allowed in this option.  I agree that a granularity of 5 might be more reasonable.

Reply:
1 and 100 are the minimum/maximum for the MB API, if you set it outside this range it'll fallback to the default (which is 25). So we should not allow the user to set such nonsense.

I agree that setting 50 or 51 doesn't make any real difference, but I also don't see any reason why we should make this more complicated than it is and artificially limit the values.

Reply:
The step approach really would be weird, especially with a limit of 1. The step just changes what the arrow buttons do, but not the selectable choices, and since it is an input you can enter every number you want (inside the range). We could handle that, but it starts getting really confusing if the values e.g. get rounded after input. It just looks random and arbitrary for the user.

But another more radical suggestion: We make it a dropdown and even more strictly limit it to a choice of just 25, 50, 75 and 100. That fulfills the main concern to give the user the ability to increase the default limit and avoids changing it to values that don't make sense (I think below 25 it starts getting pointless already). What do you think? ec8c71ab63e7ca6e3579cd045e7a842abcc07a94

![grafik](https://user-images.githubusercontent.com/29852/174729790-5bb7e8d7-1e17-4a19-90e7-939a7696832a.png)